### PR TITLE
Feat/show projects

### PR DESCRIPTION
**Pull request by MatheusLukas**

### CODE REVIEW

**Review by zKriguer — COMMENTED** [review]
Delete those images and commit again.

File: src/components/CardsProjects/CardsProjects.tsx
Comment (fellipeutaka) [inline src/components/CardsProjects/CardsProjects.tsx:1]: Rename this component to CardsProjects

**Review by fellipeutaka — CHANGES_REQUESTED** [review]

File: src/components/AttractionButton.tsx
Comment (zKriguer) [inline src/components/AttractionButton.tsx:1]: Remove console.log

File: src/components/CalButton.tsx
Comment (zKriguer) [inline src/components/CalButton.tsx:1]: Remove this console.log too

File: src/components/CardsProjects/CardsProjects.tsx
Comment (zKriguer) [reply src/components/CardsProjects/CardsProjects.tsx:1]: Updated. Closing this conversation.

**Review by fellipeutaka — APPROVED** [review]

**Pull request merged**